packages/lib/entityPermissionUtils.server.ts

```diff
@@ -1,125 +1,75 @@
+import { PermissionString } from "@calcom/features/pbac/domain/types/permission-registry";
+import { PermissionCheckService } from "@calcom/features/pbac/services/permission-check.service";
 import { MembershipRole } from "@calcom/prisma/enums";
 
-export const enum ENTITY_PERMISSION_LEVEL {
-  NONE,
-  // It is owned by user and user has write access to it
-  USER_ONLY_WRITE,
-  // All members of the team has access to it and user has read access to it
-  TEAM_READ_ONLY,
-  // All members of the team has access to it and user has write access to it
-  TEAM_WRITE,
-}
+export type EntityPermission = {
```

Comment:
nit: Better name could be _Entity_ only as it doesn't have any permission related thing. It just have required props that are needed to derive permissions